modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -715,7 +629,7 @@ protected function register_controls() {
 				'label' => esc_html__( 'Text Color', 'elementor' ),
 				'type' => Controls_Manager::COLOR,
 				'selectors' => [
-					'{{WRAPPER}}' => '--tabs-v2-title-color-hover: {{VALUE}}',
+					'{{WRAPPER}} .elementor-tab-title:hover' => '--tabs-v2-title-color-hover: {{VALUE}}',
```

Comment:
I am new to controls, what does this mean?

Reply:
@davseve The css variable is assigned when someone hover over the title of the specific instance of the widget item.